vnext/ReactUWP/Views/Image/ImageViewManager.cpp

```diff
@@ -85,6 +86,19 @@ class ImageShadowNode : public ShadowNodeBase {
 
       imageViewManager->EmitImageEvent(reactImage.as<winrt::Grid>(), succeeded ? "topLoad" : "topError", source);
       imageViewManager->EmitImageEvent(reactImage.as<winrt::Grid>(), "topLoadEnd", source);
+
+
+      // ImageView gets flipped by yoga layout in RTL mode. Use ScaleTransfrom to flip it back.
+      const auto layoutDirection = winrt::Windows::ApplicationModel::Resources::Core::ResourceContext()
+                             .GetForCurrentView()
+                             .QualifierValues()
+                             .Lookup(L"LayoutDirection");
```

Comment:
Is this the only scenario that needs to be detected?  I'm wondering what happens if someone sets direction:rtl on an ancestor of the Image.

Reply:
@kmelmon turns out our Image is not actually an Image, it's a Grid with image background instead. Grid inherits flow direction from parent and mirrors its background in RTL mode. I forced the Grid container to LTR mode, I think that fixed the issue.

Reply:
nice!  this makes sense.

---
In reply to: [378605745](https://github.com/microsoft/react-native-windows/pull/4064#discussion_r378605745) [](ancestors = 378605745)